src/components/AppHeader/AppHeader.tsx

```diff
@@ -2,6 +2,7 @@ import { useEffect } from 'react';
 import { getOauthUrl } from '@/constants';
 import { api, useGrowthbookGetFeatureValue, useOAuth } from '@/hooks';
 import { getCurrentRoute } from '@/utils';
+import Chat from '@/utils/chat';
```

Comment:
nit: can we export the chat.ts file in the index.ts utils file? then we can just do this
```suggestion
import { Chat, getCurrentRoute } from '@/utils';
```